error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -247,12 +248,13 @@ Optional<S> after(Stream<T> stream, Function<? super T, S> function) {
   /** In order to test whether a stream has any element, simply try to find one. */
   static final class StreamIsEmpty<T> {
     @BeforeTemplate
-    boolean before(Stream<T> stream) {
+    boolean before(Stream<T> stream, Collector<? super T, ?, ? extends Collection<?>> collector) {
```

Comment:
Isn't it nicer to have a separate before template for this one?

Reply:
No strong opinion. (Something we could enforce one way or another with automation...)